edge-agent/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG base_tag=1.0.5.3-linux-arm32v7
+ARG base_tag=1.0.5.4-linux-arm32v7
```

Comment:
The following images are not being updated correctly: 
```
MetricsValidator
ModuleRestarter
NumberLogger
Relayer
TestResultCoordinator
TwinTester
```

Reply:
Good catch I only updated the modules where the base images lived.

Reply:
After you do the changes, could you please rebuild the whole thing?

Reply:
I would expect the list to be more similar to this from the master PR:
```
iotedge-diagnostics-dotnet
MetricsCollector
NetworkController
CloudToDeviceMessageTester
DeploymentTester
DirectMethodReceiver
DirectMethodSender
EdgeHubRestartTester
load-gen
MetricsValidator
ModuleRestarter
NumberLogger
Relayer
TemperatureFilter
TestResultCoordinator
TwinTester
```

Reply:
Why is `NumberLogger` still using `1.0.6.2` ?

Reply:
It was probably an error missed in the last update. I will bump